src/encoded/static/components/matrix_reference_epigenome.js

```diff
@@ -647,43 +756,80 @@ class MatrixPresentation extends React.Component {
             };
         }
 
+        const rowCount = matrix => matrix.y['biosample_ontology.classification'].buckets.reduce((currentTotalCount, termName) => currentTotalCount + termName['biosample_ontology.term_name'].buckets.length, 0);
+
         return (
-            <div className="matrix__presentation">
-                <div className={`matrix__label matrix__label--horz${!scrolledRight ? ' horz-scroll' : ''}`}>
-                    <span>{context.matrix.x.label}</span>
-                    {svgIcon('largeArrow')}
+            <React.Fragment>
+                <div className="view-controls-container">
+                    <ViewControls results={this.props.context} alternativeNames={['Search list', 'Tabular report', 'Summary matrix']} />
```

Comment:
ViewControls seems like a very generic name for a component that is not generic.

Reply:
It is definitely getting more specialized all the time! I think Forrest is working on a ticket that should change a lot about how ViewControls is implemented.